docs/guide/request_lifecycle.md

```diff
@@ -0,0 +1,253 @@
+
+If the controller returns something other than `Response`, the `HttpKernel` dispatches a `kernel.view` event (`KernelEvents::VIEW`).
+In the case of a URL Alias, the controller most likely returns a ContentView.
+The `ViewRendererListener` (`ezpublish.view.renderer_listener`) uses the `ContentView` and the `TemplateRenderer` (`ezpublish.view.template_renderer`) to get the content of the `Response` and attach this new `Response` to the event.
+The `HttpKernel` retrieves the response attached to the event and continues.
+
+
+## Kernel's response event and `Response` sending
+
+The `HttpKernel` sends a `kernel.response` event (`KernelEvents::RESPONSE`). For example, if HTTP cache is used, response's headers may be enhanced.
+
+The `HttpKernel` sends a `kernel.finish_request` event (`KernelEvents::FINISH_REQUEST`). The `VerifyUserPoliciesRequestListener` (`siso_core.verify_user_policies_request_listener`) (priority 100) filters routes on its policy configuration.
+
+!!! caution "Permission control"
+ 
+     See [Permissions for routes](permissions.md#permissions-for-routes).
+
+Finally, the `HttpKernel` send the response.
+
+If an exception occurs during this chain of events, the `HttpKernel` sends a `kernel.exception` and tries to get a `Response` from its listeners.
+
+The `HttpKernel` sends the last `kernel.terminate` event (`KernelEvents::TERMINATE`). For example, the `BackgroundIndexingTerminateListener` (`ezpublish.search.background_indexer`) (priority 0) removes from the `SearchService` index possible content existing in the index but not in the database.
+
+
+## Summaries
+
+### Summary of events and services
+
+* event=`kernel.request`
+    - 45:`ezpublish.siteaccess_match_listener`
+        - `ezpublish.siteaccess_router`
+        - event=`ezpublish.siteaccess`
+            - 255:`ezpublish.siteaccess_listener`
+    - 32:`router_listener`
+        - `ezpublish.chain_router`
+            - tag=`router`
+                - `router.default`
+                - `ezpublish.urlwildcard_router`
+                - `ezpublish.urlalias_router`
+    - 16:`locale_listener`
+    - 13:`EzSystems\EzPlatformAdminUi\EventListener\RequestListener`
+* event=`kernel.controller`
+    - 10:`ezpublish.view_controller_listener`
+        - `ezpublish.view_builder.registry`
+            - tag=`ibexa.view_builder`
+                - `ezpublish.view_builder.content`
+                    - `ezpublish.view.configurator`
+* event=`kernel.controller_arguments`
+* event=`kernel.view`
+    - 0:`ezpublish.view.renderer_listener`
+        - `ezpublish.view.template_renderer`
+* event=`kernel.response`
+* event=`kernel.finish_request`
+    - 100:`siso_core.verify_user_policies_request_listener`
+* event=`kernel.terminate`
+    - 0:`ezpublish.search.background_indexer`
+
+### Examples request attributes timeline
+
+|  Event                  |  Service                              |  Request attribute  |  Example #1   |  Example #2     |
```

Comment:
The table is too wide and render in an unreadable way on mid-sized screens. Would it hurt to keep only one example column? Or redo it in some other way.